pkg/controllers/tagging/metrics.go

```diff
@@ -14,23 +14,15 @@ limitations under the License.
 package tagging
 
 import (
+	"sync"
+
 	"k8s.io/component-base/metrics"
 	"k8s.io/component-base/metrics/legacyregistry"
-	"sync"
 )
 
 var register sync.Once
 
 var (
-	workItemDuration = metrics.NewHistogramVec(
```

Comment:
are we ok dropping this metric?

Reply:
We should have an equivalent metric from the workqueue package itself via: https://github.com/kubernetes/cloud-provider-aws/blob/bf18a5157151081039ac8e9a4882df2e923fb3bd/pkg/controllers/tagging/tagging_controller.go#L35

but please make sure we actually scrape that one @kmala !

Reply:
work queue metrics are available by default https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/metrics/prometheus/workqueue/metrics.go#L40 and i had verified the same the metrics endpoint of the CCM will provide the tagging controller metrics

Reply:
Yes we've had those metrics for a while because of the import above (you have to import it for these to be registered: https://github.com/kubernetes/kubernetes/blob/d06398aac3e378da0c95472cd39713998206e9ff/staging/src/k8s.io/component-base/metrics/prometheus/workqueue/metrics.go#L26)

But do we actually scrape those? I'm pretty sure we've been scraping this other metric that we're removing

Reply:
yes, these metrics are available and i had verified the same by querying the metrics endpoint.
what do you mean by scrape those?